homekit/controller/ble_impl/__init__.py

```diff
@@ -644,6 +654,17 @@ def write(request, expected):
     return write
 
 
+def create_ble_pair_verify_write(characteristic, characteristic_id):
+    # This is a temporary wrapper until the pairing functions are also migrated to
```

Comment:
what needs to be done to handle the temporary?

Reply:
When this PR is merged I will raise a similar one for the `perform_pair_setup_part1` and `perform_pair_setup_part2` functions. At that point this wrapper will go away. It's a fairly mechanical change - if this is merged tomorrow the i will raise the follow up PR Sun evening or Mon morning.

(It's just because the transport is calling TLV.encode_list now, rather than the state machine doing it, and the BLE code path uses the same function for pair_verify and pair_setup. So using the state machine approach for both of them will mean i can get rid of the temporary function, if that makes sense).